com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HighDefinitionRenderPipelineProjectorEditor.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    [CustomEditorForRenderPipeline(typeof(Projector), typeof(HDRenderPipelineAsset))]
+    [CanEditMultipleObjects]
+    public class HighDefinitionRenderPipelineProjectorEditor : Editor
```

Comment:
Editor class are usually private/internal.

We want to only expose in the user API what have meaning for them.